docs/reference/release-notes/860.md

```diff
@@ -113,6 +113,18 @@ On Camunda 8 SaaS, you can now configure hourly backup intervals. Simply head to
 
 In Operate, the visualization of the executed path of a process instance is improved. Both executed sequence flows and flow nodes are highlighted, allowing you to display how often each flow node was executed.
 
+### Write rate limiting based on exporting rate
+
+<!-- https://github.com/camunda/product-hub/issues/1426 -->
+
+We introduce a new unified flow control mechanism inside of Zeebe that is able to limit user commands (by default it tries to achieve 200ms response times) and rate limit writes of new records in general.
```

Comment:
Non-blocking suggestion (reword):

We introduced a new Zeebe unified flow control mechanism that can limit user commands and rate limit writes of new records in general. By default, Zeebe tries to achieve 200ms response times for user commands.

(The brackets in the middle of the sentence breaks the reading flow for me)